src/core/settings.py

```diff
@@ -97,6 +97,7 @@ def cfg(path, default=0xDEADBEEF):
     # order is important here. the content response is checked
     # *after* the api v1/2 transformation (if any)
     "publisher.middleware.content_check",
+    "publisher.middleware.incompatible_v2_check",
     # v1 poa+vor are now obsolete and will be removed
     "publisher.middleware.apiv1_deprecated",  # api v1 and v2 content transformations. temporary.
```

Comment:
I take it that v1 of this Content-Type is still supported

Reply:
eh ... technically, yes, actually no. I need to open a PR on api-raml to remove it and deprecate vor v2

Reply:
https://github.com/elifesciences/api-raml/pull/239#issuecomment-517673522